integration_tests/dbt_project.yml

```diff
@@ -5,15 +5,25 @@ version: '0.1.0'
 
 profile: "integration_tests"
 
+# models:
+#   +schema: "unified_rag_{{ var('directed_schema','dev') }}"
 models:
-  +schema: "unified_rag_{{ var('directed_schema','dev') }}"
+  unified_rag:
+    +schema: unified_rag
+    intermediate:
+      +materialized: view
+    unstructured:
+      +materialized: view
+    staging:
+      +materialized: view
+      +schema: unified_rag_source
```

Comment:
Please remember to remove these before merging.

Reply:
Removed